src/Tooltip.js

```diff
@@ -88,6 +88,9 @@ class Tooltip extends React.Component<Props, State> {
             {children}
           </TouchableOpacity>
         );
+      case 'render':
+        toggleTooltip()
+        return children;
```

Comment:
I don't think it is ideal to have this side-effect on the render body. 

And this logic could be implemented on the consumer side so I'm closing this PR.